newton/examples/sensors/example_sensor_contact.py

```diff
@@ -127,29 +141,12 @@ def __init__(self, stage_path: str | None = "example_contact_sensor.usd", num_en
 
         self.solver = newton.solvers.SolverMuJoCo(self.model)
 
-        self.renderer = None
-        if stage_path:
-            self.renderer = newton.viewer.RendererOpenGL(
-                path=stage_path,
-                model=self.model,
-                scaling=2.0,
-                up_axis=str(up_axis),
-                screen_width=1280,
-                screen_height=720,
-                camera_pos=(0, 4, 30),
-            )
+        self.viewer.set_model(self.model)
 
         self.state_0 = self.model.state()
         self.state_1 = self.model.state()
         self.control = self.model.control()
 
-        self.sim_time = 0.0
-        fps = 60
-        self.frame_dt = 1.0 / fps
-
-        self.sim_substeps = 10
-        self.sim_dt = self.frame_dt / self.sim_substeps
-
         self.next_reset = 0.0
         self.step_count = 0
```

Comment:
I think you can rename this variable to something like `seed` since it is only used as the seed argument in the kernels.